turbinia/turbiniactl.py

```diff
@@ -408,6 +411,10 @@ def main():
   else:
     log.setLevel(logging.INFO)
 
+  # Enable tasks debugging for supported tasks
+  if args.debug_tasks:
+    config.DEBUG_TASKS = True
+
```

Comment:
I don't think this is going to work this way because this will set this variable on the client, but it's not going to affect the Worker, which is where this needs to take effect.   Look for where the `jobs_allowlist` is set in the `request.recipe`, and you'll probably want to add a flag for this there.  This will get plumbed through to the tasks in `evidence.config`.  For an example, see how the Plaso Task reads from `evidence.config`.

Reply:
I added it to request recipe and tested it locally and it seems to be working fine. Not sure where to add tests for recipe flags, let me know where these type of tests go.